static/src/stylesheets/module/onward/_rich-links-enhanced.scss

```diff
@@ -200,3 +200,30 @@
     float: none;
     width: auto;
 }
+
+.rich-link--paidfor {
+    &.tone-media--item,
```

Comment:
Yeah, the media tone has high(ish) specificity selectors. Not sure at the moment if we want different styles for paid galleries/paid articles so they're both the same for now.

Open to suggestions for a neater way to do this though! :)